.github/workflows/macosx.yml

```
@@ -40,5 +41,5 @@ jobs:
       - name: Report coverage to codecov
         run: |
           /Library/Developer/CommandLineTools/usr/bin/llvm-profdata merge -sparse ${{ env.LLVM_PROFILE_FILE }} -output pyxmlsec.profdata
-          /Library/Developer/CommandLineTools/usr/bin/llvm-cov show ${{ env.PYXMLSEC_LIBFILE }} -instr-profile=pyxmlsec.profdata src > coverage.txt
+          /Library/Developer/CommandLineTools/usr/bin/llvm-cov show ${{ env.PYXMLSEC_LIBFILE }} --arch=x86_64 --instr-profile=pyxmlsec.profdata src > coverage.txt
```

Is it really necessary to specify `--arch`? If so, this may be better:
```suggestion
          /Library/Developer/CommandLineTools/usr/bin/llvm-cov show ${{ env.PYXMLSEC_LIBFILE }} --arch=$(uname -m) --instr-profile=pyxmlsec.profdata src > coverage.txt
```

---

`--arch` needs to be specified, or you see this step failing [like this](https://github.com/xmlsec/python-xmlsec/actions/runs/8368326966/job/22912262609#step:9:16) with the message:
> Failed to load coverage: `-arch` specifier is invalid or missing for universal binary